### PR TITLE
LPD-27463

### PR DESCRIPTION
**Pull request by vngoliveira**
[LPD-27463](https://liferay.atlassian.net/browse/LPD-27463)

### CODE REVIEW

**Comment by marcelabc** [comment]
Just started reviewing :)

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java [hunk 1]
```diff
@@ -2805,6 +2805,28 @@ public void testGetValuesList() throws Exception {
 		_assertObjectEntryValues(29, values1, valuesList.get(0));
 		_assertObjectEntryValues(29, values3, valuesList.get(1));
 
+		// Predicate and Search
+
+		String search = "John";
```
Comment (marcelabc) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java:2810]: We don't need this new variable.
Can you inline the usages?

In the first call, I would put "John" directly.
In the second call, I would put `RandomTestUtil.randomString()`.

Okay?

**Review by marcelabc — CHANGES_REQUESTED** [review]
Nice work @vngoliveira :smile: 
I finished my review, but there were just minor changes. Let me know if you have questions :smile:

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:forward:force

**Pull request closed without merging**